### PR TITLE
Adapted test 109 to use multivalue, want to remove 4 tests, changed subids to static values

### PR DESCRIPTION
**Pull request by ScorpioBroker**
Hi,

I changed test 109 to use the multivalue instead of an array.
I removed 4 tests out of which 3 are obsolete imho. And one needs a bit more discussion.

Test 069 & 055 expects an empty entity but a 404 when you want a specific attribute and that is not a available should be the correct reply in my opinion. you are asking for a resource (that attribute of the entity) which is not available.
Test 070 expects an attribute to be missing when no correct @context is provided but you'll get a not compacted version of it. It will not be removed.
Both things are already tested in other tests so they can just be removed imho.

Test 134 expects an error on setting watchedAttributes to null.
I know this is explicitly forbidden in the spec but this shouldn't be in imho. In JSON-LD null is syntactically the same as not giving it (removed/not in). No error should be expected here. 
This doesn't change anything for other operations were null is used (since it is used there with the same syntax). So just remarking that null is not a value but means "not set" should be fine without forbidding it.
Practically JSON-LDs expand removes entries with null values. So it not being there and it being set to null is the same from JSON-LD POV. 
@jason-fox @martin-p-bauer @kzangeli 
What are you thinking about this?

I also made the subscription ids when possible static instead of by the current time. It's so much easier to track what happened.

BR
Benni

### CODE REVIEW

**Comment by ScorpioBroker** [comment]
I did one more change. I gave a bit of waiting time between the initial entity create and the subscribe. This allows a bit for async handling inside a broker. Every once in a while a test would create and subscribe so fast subsequently that the original create was not yet gone through the system and than the acc server would get 2 instead of 1 notification.

**Comment by jason-fox** [comment]
These changes make sense to me, but I'd wait for @kzangeli to check before merge.

**Review by jason-fox — APPROVED** [review]

**Pull request merged**
